complex/ft_msg.c

```diff
@@ -254,6 +254,25 @@ int ft_send_dgram(void)
 	return ret;
 }
 
+int ft_send_dgram_done(void)
+{
+        int ret,try;
+	for (try = 0; try < 1000; try++) {
+                ret = ft_send_msg();
+		if (ret)
+			return ret;
+
+		ret = ft_recv_dgram();
+		if (ret != -FI_ETIMEDOUT)
+			break;
+
+		if (test_info.caps & FI_TAGGED)
+			ft_tx.tag--;
+		ft_tx.seqno--;
+	}
+	return ret;
+}
```

Comment:
The only difference between this call and ft_sendrecv_dgram() is that this calls ft_send_msg, versus ft_send_dgram.  Why wasn't ft_sendrecv_dgram() used instead?  Especially since this code decrements seqno, which is incremented in ft_send_dgram?  Is this just a bug?